src/components/LanguageListItemCard/LanguageListItemCard.tsx

```diff
@@ -67,14 +66,14 @@ export const LanguageListItemCard: FC<LanguageListItemCardProps> = ({
             <DisplaySimpleProperty
               LabelProps={{ color: 'textSecondary' }}
               label="Ethnologue"
-              value={language?.ethnologueName?.value}
+              value={language?.ethnologue.name.value}
```

Comment:
I believe we are actually looking for `ethnologue.code` here which is a 3 digit ISO 639-3 code.
Can you add that to the label too?

Reply:
Changed